examples/prometheus.yaml

```diff
@@ -73,16 +73,55 @@ data:
       - source_labels: [__meta_kubernetes_pod_name]
         action: replace
         target_label: pod
-    - job_name: 'nginx-plus-ingress'
+    - job_name: 'nginx-ingress'
```

Comment:
Will having two jobs cause duplicate metrics? Do we need one of these to be configured to drop metrics handled by the other?

Reply:
Let me verify

Reply:
Yeah, looks like we get one entry per job. Maybe it just makes sense to leave the nginx plus scrape config, since it covers both OSS and Plus.

Reply:
I think this solution is fine. Dropping metrics would add some complexity that isn't really necessary.